hutch_python/user_load.py

```diff
@@ -50,3 +56,181 @@ def get_user_objs(load, *, ask_on_failure=True):
                 )
 
     return objs
+
+
+def update_objs(
+    obj_ns: utils.HelpfulNamespace,
+    entry: str,
+    attrs: dict,
+    fn: Callable[[BaseInterface, Union[dict, list[str]]], None]
+) -> None:
+    """
+    Helper function for updating object namespaces
+    obj_ns is a HelpfulNamespace, which may include other
+    HelpfulNamespace's.  As a result this must be semi-recursive.
+
+    Parameters
+    ----------
+    obj_ns : HelpfulNamespace
+        Namespace of all objects
+
+    entry : str
+        Name of device to edit
```

Comment:
Oh, is this not just a name of a device, e.g. it can be a class name? Is that what I'm confused in my other comments?